src/commcare_cloud/ansible/roles/backups/tasks/main.yml

```diff
@@ -1,29 +1,16 @@
-- name: Create aws config directory
+- name: Set up aws credentials
   become: yes
-  file:
-    path: '~{{ blobdb_user }}/.aws'
-    owner: "{{ blobdb_user }}"
-    group: "{{ blobdb_user }}"
-    mode: 0700
-    state: directory
+  import_role:
+    name: backups
+    tasks_from: set_up_aws_credentials.yml
+  vars:
+    s3_user: "{{ blobdb_user }}"
+    s3_user_home_dir: "~{{ blobdb_user }}"
```

Comment:
what does `~{{ blobdb_user }}` resolve to?

Reply:
It resolves to `~root` which is normally /var/root` I think